content/_getting-started/executions page.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Executions Page
+layout: article
+section: Basic Concepts
+order: 1
+---
+
+This document provides basic information on [Executions](#executions) and [Executions page](#executions-page).
+
+## Executions
+An Execution is the path of an initial message through a Flow. As the message gets into the Flow (generated by a scheduler, received by a webhook, etc.), it is assigned an Execution ID. This ID follows the message through the entire Flow, forming an Execution. Also, the same ID will be assigned to any messages caused by this initial message in the Flow. The following diagram demonstrates Executions using a Webhook flow:
+
+![Executions page filters](/assets/img/getting-started/exec-page/exec1.png)
+
+A client sent two messages to a Webhook URL simultaneously. The A message was passed by the Webhook trigger to the next action as is. While processing the message the action produced 10 more messages which were passed to the following action. In summary we can say that an external stimulus produced an A Execution of 11 messages being processed in the platform. Similarly another external stimulus produced the B Execution with 11 messages. These two Executions are separated logically but not necessarily physically.
+
+
+## Executions Page
+The page is, basically, a list of all the Executions, which can be filtered by:
+
+- Date
+
+- Successful/Erroneous
+
+- Flows  
+
+If new executions are added to the list while you are browsing it, you will get a message at the top of the list, and a prompt to load them.
```

Comment:
Explain more details on the prompt. Why it is there? Why don't we just prepend the new executions on top of the table. There is also a `X` in the prompt. Explain what happens if you click on that.